lib/geordi/util.rb

```diff
@@ -84,12 +84,14 @@ def binstub_or_fallback(executable)
       end
 
       def console_command(environment)
+        multiline_flag = irb_version >= Gem::Version.new('1.2') ? '-- --nomultiline' : ''
```

Comment:
Please also set an upper limit, as per [this card](https://makandracards.com/makandra/490022-how-to-fix-pasting-in-irb-1-2+-is-very-slow) the issue is fixed in 1.2.6.

Tip:
```
multiline_flag = if condition
  ...
end
```
will leave the variable with `nil` if the condition is `false`. It can be used right ahead in interpolation like `"console #{multiline_flag}"`.